presto-tpch/src/main/java/com/facebook/presto/tpch/TpchColumnHandle.java

```diff
@@ -27,14 +30,22 @@
 {
     private final String columnName;
     private final Type type;
+    private final Optional<List<SubfieldPath>> subfieldPaths;
```

Comment:
Can we represent emptiness with the list?  Empty list rather than Optional.empty